### PR TITLE
ToDo List - Add & remove function

### PR DESCRIPTION
**Pull request by mgmediaweb**
# Features

- Make the add task function.
- Validate the field with in-build required.
- Use Event Listener to form send.
- Use class and specific methods to set/get into the localStorage.
- The Remove function call to method into the class to remove LocalStorage.
- Re-list the tasks on the screen after the add/remove method.

## Live demo:
https://mgmediaweb.github.io/webpack/dist/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## Live Demo
 
-[Live Demo Link](https://mgmediaweb.github.io/webpack)
+[Live Demo Link](https://mgmediaweb.github.io/webpack/dist/)
```
Comment (brytebee) [inline README.md:17]: - [ ] **Required:**

https://github.com/mgmediaweb/webpack/blob/086e8be54ed4d7b66b75859a9bce67d5d4566524/src/todoClass.js#L73-L83

When this method is called on clear all with multiple items or items are deleted from the mid of the list, the re-order behaves strangely.

re-order

Kindly fix this!

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @mgmediaweb👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- ✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mgmediaweb) [reply README.md:17]: You are absolutely right, I just uploaded the update repairing that

**Review by topeogunleye — APPROVED** [review]
### Hi @mgmediaweb, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**